Stellar-reviewable-request-asset.x

```diff
@@ -27,13 +33,18 @@ struct AssetCreationRequest {
     ext;
 };
 
+//: AssetUpdateRequest is used to update asset with provided parameters
 struct AssetUpdateRequest {
-	AssetCode code;
+    //: Code of asset to update
+    AssetCode code;
+    //: Arbitrary stringified JSON object that can be used to attach data to be reviewed by the admin
     longstring creatorDetails; // details set by requester
-	uint32 policies;
+    //: New policies to set, will override existing ones
+    uint32 policies;
+    //: Used to keep track of rejected requests update. On each rejected AssetUpdateRequest update, sequenceNumber increases
```

Comment:
//: Used to keep track of rejected requests update (`SequenceNumber` increases after each rejected AssetUpdateRequest update).